src/profparser.cpp

```diff
@@ -71,7 +68,7 @@ void ProfileParser::parse(const std::string &path, std::ifstream &fs) {
     }
 }
 
-void ProfileParser::parseExtraLine(const std::string &line, std::function<void(const std::string_view)> printBadLine) {
```

Comment:
Do you still need `#include <functional>` now in `profparser.h` then?

Reply:
Indeed. I checked all includes and found another unused one. Pushed a commit to clean this up.